models/db.js

```diff
@@ -34,6 +34,12 @@ db.categoryModel = require('../models/category')(sequelize, Sequelize);
 db.productModel = require('../models/product')(sequelize, Sequelize);
 db.productImageModel = require('../models/product_image')(sequelize, Sequelize);
 db.cartModel = require('../models/cart')(sequelize, Sequelize);
+db.addressModel = require('../models/address')(sequelize, Sequelize);
+db.orderModel = require('../models/order')(sequelize, Sequelize);
+db.order_productModel = require('../models/order_product')(
```

Comment:
please use same naming convention in whole project
because in orderModel  and order_productModel use underscore